lti_tool/templates/lti_tool/lti_submission.html

```diff
@@ -10,11 +10,8 @@
 {% block siblings_bottom %}{% endblock %}
 {% block footer %}{% endblock %}
 
-{% block exercise_wait %}
-{% include "lti_tool/_lti_exercise_wait.html" %}
-{% endblock %}
-
 {% block exerciseinfo %}
 {{ block.super }}
 {% include 'exercise/_submission_info.html' %}
+{% include "lti_tool/_lti_exercise_wait.html" %}
```

Comment:
I don't think `_lti_exercise_wait.html` belongs to the `exerciseinfo` block. The `exerciseinfo` block is intended for the column on the right side of the exercise and submission pages.

Block `exercise_wait` is confusing since it is not defined at all in `exercise_base.html` or `exercise.html`. The purpose of the block `exercise_wait` needs to be checked. Perhaps, we will remove it completely also from `submission.html`.